.gitignore

```diff
@@ -17,3 +17,14 @@ project.lock.json
 /.dotnet/
 tools/**/obj/
 tools/**/build/
+
+# Docs artifacts
+docs/obj/
+docs/bin/
+
+# Xml doc/test files
+*.xml
+src/Castle.Core/Castle.Core.xml
+src/Castle.Services.Logging.NLogIntegration/Castle.Services.Logging.NLogIntegration.xml
+src/Castle.Services.Logging.SerilogIntegration/Castle.Services.Logging.SerilogIntegration.xml
+src/Castle.Services.Logging.log4netIntegration/Castle.Services.Logging.log4netIntegration.xml
```

Comment:
Oh, don't these usually get written to the output `bin` directory next to the DLL and PDB files? That is what VS used to default to before the new CLI tooling.

Reply:
I was almost sure that [OutputPath](https://github.com/castleproject/Core/blob/master/src/Castle.Core/Castle.Core.csproj#L22) was the way to do it. But it isn't working here for some reason.

Reply:
I'm not sure what is different to Windsor because Windsor 4.1.0 has XML files for each target framework in their bin/debug or bin/release and in the NuGet packages, and it doesn't specify the `DocumentationFile` element.

We need to fix this because I just realised that if the file is getting written here than wouldn't that mean that the last target to get built becomes the output for all targets, i.e. if netstandard1.6 is built last than that's the docs for all targets.

Reply:
Will have a fix for this soon.